tracker/enqueue_test.go

```diff
@@ -333,15 +333,6 @@ func TestBadObjectReferences(t *testing.T) {
 			Name:       "kaniko",
 		},
 		match: "Kind",
-	}, {
-		name: "Missing Namespace",
```

Comment:
could you please add a test into `TestAllowedObjectReferences` with the lack of a namespace?